lib/ae_mdw/db/contract.ex

```diff
@@ -143,7 +143,7 @@ defmodule AeMdw.Db.Contract do
 
   @spec logs_write(state(), txi(), txi(), Contract.call()) :: state()
   def logs_write(state, create_txi, txi, call_rec) do
-    contract_pk = :aect_call.contract_pubkey(call_rec)
+    {_id_tag, contract_pk} = call_rec |> :aect_call.contract_id() |> :aeser_id.specialize()
```

Comment:
This wouldn't be needed since the `:aect_call` has the `contract_pubkey` getter.

Reply:
You're right, done